client/src/pages/Login/LoginForm/useStyles.ts

```diff
@@ -7,9 +7,10 @@ const useStyles = makeStyles((theme: Theme) => ({
     marginTop: theme.spacing(1),
   },
   label: {
-    fontSize: 19,
-    color: 'rgb(0,0,0,0.4)',
-    paddingLeft: '5px',
+    color: '#000000',
+    fontSize: 12,
```

Comment:
Similar to the component above about the usage of MUI's theme, we can set font sizes for variants within the theme and apply these variants to our `Typography` components to ensure a consistent font size across the app

Ditto where applicable

Reply:
Done the changes